packages/teleterm/src/ui/DocumentGateway/DocumentGateway.tsx

```diff
@@ -101,3 +104,48 @@ export function DocumentGateway(props: State) {
     </Box>
   );
 }
+
+function CliCommand({
+  cliCommand,
+  onClick,
+}: {
+  cliCommand: string;
+  onClick(): void;
+}) {
+  return (
+    <Flex
+      p="2"
+      alignItems="center"
+      justifyContent="space-between"
+      borderRadius={2}
+      bg={'primary.dark'}
+      mb={4}
+    >
+      <Flex
+        mr="2"
+        style={{
+          overflow: 'auto',
+          whiteSpace: 'pre',
+          wordBreak: 'break-all',
+          fontSize: '12px',
+          fontFamily: useTheme().fonts.mono,
```

Comment:
Minor thing, you can use css prop - it has access to theme (and for me, syntax of `css` is nicer than of `style` :) )
```
css={{
          overflow: auto,
          white-space: pre,
          word-break: break-all,
          fonts-size: 12px,
          font-family: ${props => props.theme.fonts.mono},
        }}
```

Reply:
That's indeed much better, thanks!